data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import os
+import psycopg2
+import getpass
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, Title TEXT, Description TEXT, Company TEXT, Location TEXT, SalaryMin INTEGER, PostURL TEXT)")
+# con.commit()
+
+# make ZipRecruiter API call
+API_KEY = os.getenv("ZIPRECRUITER_API_KEY")
+if API_KEY != None:
+  try:
+    req = requests.get("https://api.ziprecruiter.com/jobs/v1?search=internship%20Job&days_ago=1&page=1&api_key=" + API_KEY)
```

Comment:
Should we be looping through all the pages to get every result?

Reply:
I'm not using the jobs_per_page parameter, which based on a few test API calls would just return all the jobs on a single page. So I'll remove the page=1 too.